dockerfiles/Dockerfile.node

```diff
@@ -1,5 +1,12 @@
-FROM node:8-stretch-slim
+FROM node:8-alpine
```

Comment:
Sadly, we have to use Debian and not Alpine because of how VSCode support docker ([note in the install section](https://code.visualstudio.com/docs/remote/containers#_getting-started)). It's still in review on Foundation but should be merged soon.

Reply:
OK. I switched to alpine because it comes with some of the image processing deps that Wagtail requires. It should be possible to install those manually in the dockerfile though.

Reply:
I think it's safe to remove them: I can run the foundation site on debian without any extra dependencies, both back and front.